package.json

```diff
@@ -95,8 +95,9 @@
     "iana-tz-data": ">=2017.0.0",
     "matchdep": "1.0.1",
     "mocha": "^3.4.2",
+    "qunit": "2.12.0",
     "semver": "^5.3.0",
-    "zoned-date-time": "1.0.0"
+    "zoned-date-time": "https://github.com/Krinkle/zoned-date-time#bbf87839ef7e8b8dfb8ceec94b2f5bad3aaef7b6"
```

Comment:
This seems unrelated to QUnit change. Please, do you mean it?

Reply:
Yes and no. See point 2 in the pull description. It is needed for it to pass. But I do not recommend merging it in this way, instead we should probably land https://github.com/rxaviers/zoned-date-time/pull/3 first and release that, then you or I can update the version number here.

Reply:
Thanks, `zoned-date-time@1.1.0` released.

Reply:
I've updated the PR.